docs/topics/security.txt

```diff
@@ -20,7 +20,7 @@ by the user's browser. However, XSS attacks can originate from any untrusted
 source of data, such as cookies or web services, whenever the data is not
 sufficiently sanitized before including in a page.
 
-Using Django templates protects you against the majority of XSS attacks.
+Using Django templates protects you against many types of XSS attacks.
```

Comment:
This is slightly related: There's a CSP section right after.. so wanted to downplay the rhetoric a bit here.